src/types.ts

```diff
@@ -109,7 +109,7 @@ export class InteractChoice {
         /**
          * The version of the transaction.
          */
-        public readonly transactionVersion: Numeric
+        public transactionVersion: Numeric
```

Comment:
Can't this stay `readonly`? The object generated with `Object.assign` shouldn't be of type `InteractChoice`, right?